dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/HdfsExtractionTask.java

```diff
@@ -44,12 +48,14 @@
 public class HdfsExtractionTask extends AbstractTask<Void> implements HdfsExtractionDumpFormat {
   private static final Logger LOG = LoggerFactory.getLogger(HdfsExtractionTask.class);
 
-  private final int poolSize;
+  private final int threadPoolSize;
+  private final String hdfsScanRootPath;
 
   HdfsExtractionTask(@Nonnull ConnectorArguments args) {
     super(HdfsFormat.ZIP_ENTRY_NAME);
     Preconditions.checkNotNull(args, "Arguments was null.");
-    poolSize = args.getThreadPoolSize();
+    threadPoolSize = args.getThreadPoolSize();
```

Comment:
I know, it's really unexpected usage of our tool and not part of the current task scope, but we already modify the line of the code, so can we add validation of input parameter here?
Something like this method `com.zaxxer.hikari.HikariConfig#setMaximumPoolSize` does.

Reply:
Done